tests/minifier.js

```diff
@@ -1813,6 +1813,18 @@ QUnit.test('Ignore custom fragments', function(assert) {
     collapseWhitespace: true,
     trimCustomFragments: true
   }), output);
+
+  input = 'foo <WC@bar> baz moo </WC@bar> loo';
+  assert.equal(minify(input, {
+    collapseWhitespace: true,
+    ignoreCustomFragments: [
+      /<(WC@[\s\S]*?)>(.*?)<\/\1>/
+    ]
+  }), input);
+  output = 'foo<wc @bar>baz moo</wc>loo';
```

Comment:
This is the case *without* `ignoreCustomFragments` - the case you are interested in is a line above, when nothing's changed, i.e. expects to equal to `input` 😉